client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInput.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {Dispatch, SetStateAction} from 'react';
+import {Airport, City, Country} from "../../ExposuresAndFlights";
+import useFlightsInvestigation from "../../useFlightsInvestigation";
+import AutocompletedField from "./AutoCompletedField/AutocompletedField";
+import {TextField} from "@material-ui/core";
+import useFormStyle from 'styles/formStyles';
+
+interface AirportInputProps {
+    airport: Airport | null;
+    setAirport: Dispatch<SetStateAction<Airport | null>>;
+};
+
+const AirportInput = ({airport, setAirport}: AirportInputProps) => {
+    const [selectedCountry, setSelectedCountry] = React.useState<Country | null>(null);
+    const [selectedCity, setSelectedCity] = React.useState<City | null>(null);
+
+    // search states
+    const [cityPrefix, setCityPrefix] = React.useState<string>('');
+    const [airportPrefix, setAirportPrefix] = React.useState<string>('');
+
+    const {countries, cities, airports,} = useFlightsInvestigation({
+        selectedCountry,
+        selectedCity,
+        cityPrefix,
+        airportPrefix,
+    });
+
+    const classes = useFormStyle();
+
+    const onCountrySelect = (event: React.ChangeEvent<{}>, newValue: any | null) => {
+        setSelectedCountry(newValue);
+    };
+
+    const onCitySelect = (event: React.ChangeEvent<{}>, newValue: City | null) => {
+        setSelectedCity(newValue);
+        newValue && onCountrySelect(event, newValue.country);
+    };
+
+    const onAirportSelect = (event: React.ChangeEvent<{}>, newValue: Airport | null) => {
+        setAirport(newValue);
+        newValue && onCitySelect(event, newValue.city);
+    };
+
+    const onCityInput = (event: React.ChangeEvent<{}>,
+                         newInputValue: string,) => {
+        console.log('cityPrefix', cityPrefix);
+        setCityPrefix(newInputValue);
+    };
+
+    const onAirportInput = (event: React.ChangeEvent<{}>,
+                            newInputValue: string,) => {
+        setAirportPrefix(newInputValue);
+    };
+
+    // Fields are temporarily text inputs only
+    // Autocomplete fields will be added when api is ready
+    const AutocomplteFields = () =>
+        <>
+            <AutocompletedField
+                constOptions
+                label='מדינה'
```

Comment:
youre using this string and the others twice, so maybe put them in a const?

Reply:
only one of them will be used :) the second one will be removed when api is resolved